CMakeLists.txt

```
@@ -17,16 +17,18 @@ include(unifex_flags)
 add_subdirectory(source)
 
 if(BUILD_TESTING OR UNIFEX_BUILD_EXAMPLES)
+  # We enable testing in both, either if doing tests of build examples
```

Can you add to this comment _why_ we need to `enable_testing()` if we're not `BUILD_TESTING`?

Also, there's a small typo:
```suggestion
  # We enable testing in both, either if doing tests or build examples
```